src/app/server/src/server.ts

```diff
@@ -0,0 +1,48 @@
+import {Request, Response} from 'express';
+
+const express = require("express");
+const path = require("path");
+const fs = require("fs");
+const hbs = require("hbs");
+
+const app = express();
+const port = process.env.PORT || 3000;
+const rootDir = path.join(__dirname, "..");
+
+app.use(express.static(path.join(rootDir, "public")));
+app.use("/files", express.static(path.join(rootDir, "config/files")));
+
+app.set("view engine", "hbs");
+app.set("views", path.join(rootDir, "views"));
+
+hbs.registerHelper('json', function(context: (...args: any[]) => any) {
+    return JSON.stringify(context);
+});
+
+const getAppConfig = (appName: string) => {
+    const filename = path.join(rootDir, "config",  `${appName}.config.json`);
+    if (fs.existsSync(filename)) {
+        const configText = fs.readFileSync(filename, {encoding: "utf-8"});
+        return JSON.parse(configText);
+    }
+    return null;
+};
+
+app.get("/", (req: Request, res: Response) => {
```

Comment:
There's no 404 handler on a non-app page (so `http://localhost:3000/not-here` returns generic page with `Cannot GET /not-here`)

Reply:
Not yet!